src/Microsoft.DotNet.CodeAnalysis/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "dependencies": {
+    "Microsoft.Net.Compilers.NetCore": "1.2.0-beta1-20160202-02",
```

Comment:
The library packages do support .NET Core so I would have expected them to be the same. I can understand using this package for the csc commandline tool but I wouldn't expect to be necessary for the analyzers.